tests/mock_tables/dbconnector.py

```diff
@@ -6,7 +6,49 @@
 import mockredis
 import swsssdk.interface
 from swsssdk.interface import redis
-
+from swsssdk import SonicV2Connector
+from swsssdk import SonicDBConfig
+import importlib
+
+def clean_up_config():
+    # set SonicDBConfig variables to initial state 
+    # so that it can be loaded with single or multiple 
+    # namespaces before the test begins.
+    SonicDBConfig._sonic_db_config = {}
+    SonicDBConfig._sonic_db_global_config_init = False
+    SonicDBConfig._sonic_db_config_init = False
+
+def load_namespace_config():
+    # To support testing single namespace and multiple
+    # namespace scenario, SonicDBConfig load_sonic_global_db_config
+    # is invoked to load multiple namespaces to support multiple
+    # namespace testing.
+    clean_up_config()
+    SonicDBConfig.load_sonic_global_db_config(
+        global_db_file_path=os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), 'database_global.json'))
+
+def load_database_config():
+    #load local database_config.json for single namespace test scenario
+    clean_up_config()
+    SonicDBConfig.load_sonic_db_config(
+        sonic_db_file_path=os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), 'database_config.json'))
+
+def init_SonicV2Connector(self, use_unix_socket_path=False, namespace=None, **kwargs):
+    self.namespace = namespace
+    self.use_unix_socket_path = use_unix_socket_path
+    for db_name in SonicDBConfig.get_dblist(self.namespace):
+        setattr(self, db_name, db_name)
+    ns_list = SonicDBConfig.get_ns_list()
+    # In case of multiple namespaces, namespace string passed to
+    # SonicV2Connector will specify the namespace or can be empty.
+    # Empty namespace represents global or host namespace.
+    if len(ns_list) > 1 and namespace == "":
+        kwargs['namespace'] = "global_db"
```

Comment:
>global_db [](start = 31, length = 9)

This is totally internal, and we can use empty string to represent 'global' namespace.

Reply:
namespace="" can represent global namespace for single asic/single namespace platform.
So if the number of namespaces is more than one, then namespace="" represents global_db.